### PR TITLE
Document External Data Feature

### PR DESCRIPTION
**Pull request by alexfornuto**
WIP to document external data sources in Pomerium Console

One of several PRs for #9 
Resolves https://github.com/pomerium/docs/issues/72

### CODE REVIEW

File: content/docs/reference/reference.json [hunk 1]
```diff
@@ -168,6 +168,52 @@
...
+    "id": "ext-data-url",
+    "title": "External Data URL",
+    "short_description": "Path to the external dataset",
+    "path": "/../enterprise/reference/configure#url",
+    "enterpriseOnly": true
+  },
+  "ext-data-record-type": {
+    "id": "ext-data-record-type",
+    "title": "External Data Record Type",
+    "short_description": "Overriden by the directory structure of an archive file dataset",
```
Comment (alexfornuto) [inline content/docs/reference/reference.json:181]: @cmo-pomerium these short descriptions appear in the Console:
image

The `path` value refers to anchors also added in this PR in `configure.md`

**Review by calebdoxsey — APPROVED** [review]

File: content/docs/enterprise/reference/configure.md [hunk 2]
```diff
@@ -209,6 +209,78 @@ A [Namespace][namespace-concept] is a collection of users, groups, routes, and p
...
+- CSV file, where the first row indicates the field names and subsequent rows are records. One of the fields **must** be an `id`.
+
+  ```json title="example CSV"
+  id,user.id
+  id1@example.com,user1
+  id2@example.com,user2
+  id3@example.com,user3
+  ```
+
+- A tar or zip file containing files of one of the formats above. The file path within the tar file specifies the record type, if not defined in the configuration. For example, in an arhive containing the following structure:
```
Comment (cmo-pomerium) [inline content/docs/enterprise/reference/configure.md:241]:
```suggestion
- A tar or zip file containing files of one of the formats above. The file path within the tar file specifies the record type if not defined in the configuration. For example, in an archive containing the following structure:
```

Spellcheck. No strong feelings about the comma, personal choice.

Comment (alexfornuto) [reply content/docs/enterprise/reference/configure.md:241]: Good call on the typo. I'mma opt to keep the comma.

File: content/docs/enterprise/reference/configure.md [hunk 3]
```diff
@@ -209,6 +209,78 @@ A [Namespace][namespace-concept] is a collection of users, groups, routes, and p
...
+
+#### Headers
+
+Headers defined here will be used when connecting to the external data source.
+
+#### Allow Insecure TLS
+
+If set, allows the import of external data from sources using untrusted TLS certificates.
+
+#### Poling Min/Max Delay
```
[the remark this answers is not in the document]
Comment (alexfornuto) [reply content/docs/enterprise/reference/configure.md:274]: Yup, just a typo :)

**Pull request merged**